content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -168,19 +138,31 @@ to manage the lifecycle of Istio.
 
 1. Install the `istio` chart:
 
-    {{< text bash >}}
+    * For a production environment use the recommended Helm values or customize as needed:
 
+    {{< text bash >}}
     $ helm install istio --name istio --namespace istio-system
+    {{< /text >}}
+
+    * To try the `bookinfo` demo, use Helm values customized for the `bookinfo` application:
 
+    {{< text bash >}}
+    $ helm install istio --name istio --values install/kubernetes/helm/istio/values-istio-demo.yaml --namespace istio-system
     {{< /text >}}
 
-1. Uninstall steps:
+## Uninstall
+
```

Comment:
thanks @andraxylia - I had a PR in progress (and in the queue) for this work, but my lab is down and nonfunctional so my PR is stranded on gear I can't connect to :)